OpenKeychain/src/main/java/org/sufficientlysecure/keychain/ui/CreateKeyFinalFragment.java

```diff
@@ -328,6 +329,9 @@ public SaveKeyringParcel createOperationInput() {
             @Override
             public void onCryptoOperationSuccess(EditKeyResult result) {
 
+                BackupNotificationUtil.setBackupReminder(getContext(),result.mMasterKeyId,
```

Comment:
`result.mMasterKeyId` might be `null` at this point. this is probably better kept in `finishWithResult`